### ISSUE TITLE
Generated pdf contains spaces

### ISSUE THREAD

**Issue by faizanm324** [Aymkdn/html-to-pdfmake#159]
So I am getting the extra spaces in the generated output, maybe dues to some specific html tags that I use!

![html-pdfmake](https://user-images.githubusercontent.com/118271663/201937659-ab71f2bc-90d8-4bea-9212-4b1042860d3e.png)

Following is the html I used
`<code style="color: gray">&nbsp;</code><code style="color: gray">&lt;Tag1&gt;</code><br/><code style="color: gray">&nbsp;</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: gray">&lt;child1 <font color=gray>equalAttr2=</font><font color=gray>"</font><font color=red>ValDiff</font><font color=#4cbb17>DiffVal</font><font color=gray>"</font> <font color=gray>equalAttr1=</font><font color=gray>"ValEqual"</font> <font color=red>deleteAttr="DelVal"</font> <font color=#4cbb17>addAttr="addVal"</font>&gt;</code><br/><code style="color: gray">&nbsp;</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: gray">&lt;/child1&gt;</code><br/><code style="color: gray">&nbsp;</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: gray">&lt;equalTagWithTextDiff&gt;</code><br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<code><font color=red>T</font><font color=#4cbb17>t</font><font color=gray>ext </font><font color=red>N</font><font color=#4cbb17>n</font><font color=gray>ot </font><font color=red>E</font><font color=#4cbb17>e</font><font color=gray>qual</font></code><br/><code style="color: gray">&nbsp;</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: gray">&lt;/equalTagWithTextDiff&gt;</code><br/><code style="color: gray">&nbsp;</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: gray">&lt;equalTag&gt;</code><br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<code><font color=gray>EqualText</font></code><br/><code style="color: gray">&nbsp;</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: gray">&lt;/equalTag&gt;</code><br/><code style="color: red">-</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: red">&lt;deleteTag <font color=red>deleteAttr1="DelVal1"</font> <font color=red>deleteAttr2="DelVal2"</font>&gt;</code><br/><code style="color: #4cbb17">+</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: #4cbb17">&lt;addTag <font color=#4cbb17>addAttr1="AddVal1"</font> <font color=#4cbb17>addAttr2="AddVal2"</font>&gt;</code><br/><code style="color: #4cbb17">+</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: #4cbb17">&lt;/addTag&gt;</code><br/><code style="color: red">-</code>&nbsp;&nbsp;&nbsp;&nbsp;<code style="color: red">&lt;/deleteTag&gt;</code><br/><code style="color: gray">&nbsp;</code><code style="color: gray">&lt;/Tag1&gt;</code><br/>`

**Comment by Aymkdn** [comment]
> maybe dues to some specific html tags that I use!

Yes, indeed. Your HTML code is overcomplicated… You should try to simplify it.

**Comment by faizanm324** [comment]
Unfortunately, I cannot simplify it as I have to generate the proper xml representation as shown in html preview!
